pkg/cmd/server/server.go

```diff
@@ -717,7 +725,8 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 		cmd.CheckError(err)
 
 		restoreController := controller.NewRestoreController(
-			s.namespace,
+			// Empty namespace so that the controller is able to retrieve Restores from any namespace.
```

Comment:
So Velero Server and API objects are not all created within a default "velero" namespace, instead they are created within the cluster namespace? Or another namespace?

Reply:
Ahh, a few lines down it is explained. Ok. Backups are deployed to the origin namespace.